streamer/src/quic.rs

```diff
@@ -717,7 +720,7 @@ pub fn spawn_simple_qos_server(
 
     let simple_qos = Arc::new(SimpleQos::new(
         qos_config,
-        quic_server_params.max_connections_per_unstaked_peer,
+        quic_server_params.max_connections_per_staked_peer,
         quic_server_params.max_staked_connections,
```

Comment:
Made PR https://github.com/anza-xyz/agave/pull/9078 for this. Needed to unblock the test here.